data/json/itemgroups/Weapons_Mods_Ammo/guns.json

```diff
@@ -273,6 +277,7 @@
       { "item": "bh_m89", "prob": 20, "charges-min": 0, "charges-max": 7 },
       { "item": "bfg50", "prob": 5, "charges-min": 0, "charges-max": 1 },
       { "item": "carbine_flintlock", "prob": 140 },
+      { "item": "famas", "prob": 40, "charges-min": 0, "charges-max": 30 },
```

Comment:
I looked, but couldn't find any info about US police/military using FAMAS. Would have to be much more rare than that (probably just set probability to 1 for now) unless you can find sources saying otherwise.

Reply:
The AN-94 on that list has the same probability and was also never used in the US. Since there is no data on civilian sales for obvious reasons, that is the comparison I went with. In either case, while lowering the probability might be fine, 1 seems far too low. This is on the obscure list, so it already won't show up a lot. As it stands, there are people who have never even found an AKM, which has probability 100 on that list, nevermind any of the rarer ones. I would suggest no lower than 20 for this.

Reply:
You can't use the AN-94's presence to justify the FAMAS.

However, at least 1 FAMAS is actually in country.

Reply:
Are the other weights being adjusted as well, then? Because right now it seems a bit of a mess.

It's also not in military/police, it's in obscure.

Reply:
Changed the probability. And various other things.